locale/it/about/index.md

```diff
@@ -20,27 +24,39 @@ const server = http.createServer((req, res) => {
 });
 
 server.listen(port, hostname, () => {
-  console.log(`Server in esecuzione all'indirizzo http://${hostname}:${port}/`);
+  console.log(`Server running at http://${hostname}:${port}/`);
 });
 ```
-Ciò è in contrasto con il modello comune del giorno d'oggi, in cui vengono sfruttati i thread del sistema operativo. Il networking basato sui thread è piuttosto inefficiente e molto difficile da usare. Inoltre, gli utenti di Node sono liberi da preoccupazioni di processi che entrano in dead-lock, dal momento in cui non ci sono blocchi. Quasi nessuna funzione in Node esegue direttamente operazioni di I/O, perciò il processo non si blocca mai. Dal momento che nulla si blocca, è molto facile sviluppare sistemi scalabili in Node.
 
-Se qualcosa in questa terminologia ti è poco familiare, qui c'è un articolo completo su [Bloccante vs Non Bloccante][]. 
+Ciò è in contrasto con il modello di concorrenza più comune di oggi in cui vengono utilizzati 
+i thread. Il networking basato su thread è relativamente inefficiente e 
+molto difficile da utilizzare. Inoltre, gli utenti di Node sono liberi da preoccupazioni di 
```

Comment:
Please replace all occurrences of Node with Node.js. It's the official way to refer to it.